### PR TITLE
WRN-8023: Import custom skin.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added a button to import a skin that conforms to our standards.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-8023

### CODE REVIEW

File: sandstone/custom-skin/src/components/ImportSkin.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		showFile: async (ev, {setColors}) => {
+			ev.preventDefault();
+			const reader = new window.FileReader();
+			reader.onload = async (event) => {
+				let text = (event.target.result).split('\n\t');
+				text.shift();
+				text.pop();
+				text = text.filter(string => {
+					// console.log(string);
+					// console.log(string.includes('/* Skin Name'));
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/components/ImportSkin.js:21]: remove these

File: sandstone/custom-skin/src/utils.js [hunk 2]
```diff
@@ -97,8 +97,26 @@ const checkColors = (arr1, arr2) => {
...
+		let colorSets = colors.map(color => color.split(':'));
+		colorSets = colorSets.map(colorSet => [colorSet[0], colorSet[1].split(';')[0].replace(/ /g, '')]);
+		colorSets[0][1] = colorSets[0][1].split('*/')[0];
+
+		return colorSets;
+	} catch (err) {
+		// eslint-disable-next-line
+		console.log(err);
+		// eslint-disable-next-line
+		window.alert('Wrong type of file!');
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/utils.js:111]: maybe we can show a sandstone Alert component in this case. WDYT?

File: sandstone/custom-skin/src/components/ImportSkin.js (hunk 1 above)
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/components/ImportSkin.js]: Done

File: sandstone/custom-skin/src/utils.js (hunk 2 above)
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/utils.js]: Done.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**